include/drivers/irq/am335x.h

```diff
@@ -11,9 +11,7 @@
 #include <machine/interrupt.h>
 #include <armv/machine.h>
 
-enum irqNumbers {
-    irqInvalid = 255
-};
+irq_t irqInvalid = 255;
```

Comment:
Make it const also everywhere

```suggestion
const irq_t irqInvalid = 255;
```